### ISSUE TITLE
Can't disable primary subtitles on Samsung TV

### ISSUE THREAD

**Issue by The-Choo-Choo-Shoe** [Stremio/stremio-features#690]
I am running Stremio Samsung TV Theater 1.2.0 on Tizen 6.5.

On other platforms there is an option to disable primary/secondary subtitles but on the Tizen OS version you don't have the option to disable primary subtitles so every new video you watch will load with subtitles enabled.

I also noticed that it doesn't remember your subtitle setting for each show, on Android TV it will remember if I disabled a subtitle for a show but on Tizen OS it does not.

**Comment by Vondanta** [comment]
Can confirm this is an issue with Theater v1.0.1 as well

**Comment by sbriongos** [comment]
+1

**Comment by jayseventwo** [comment]
+1

**Comment by quacker1982** [comment]
Yeah this is rather frustrating, hopefully a fix can be pushed at some stage.

**Comment by FatestSquirrel** [comment]
Looking forward to the 2 year anniversary of this still being a bug/*missing feature.
